### PR TITLE
Adds loincloths.

### PR DESCRIPTION
**Pull request by weesnaw-ops**
Re-adds loincloths, editing their sprite so they don't look like 'diapers' or whatever.

## About The Pull Request

This pull request re-adds loincloths as craftable and loadout-able items as default. The original reason for removal was that it looked like diapers, but the reason it was accepted was only because the owner of the code submitted the PR. It shouldn't mess with the adam and eve or gear harness code that was already put in.

This code includes updated sprites which make the loincloths significantly longer so it looks less like weird diapers.

## How This Contributes To The Skyrat Roleplay Experience

People used the loincloths, and as it was said by the coder who approved the merger, they'd rather just have the sprite changed than it removed. I know I did. 

## Changelog

:cl:
add: readds loincloths with new sprite
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/clothing/under/costumes.dm [hunk 1]
```diff
@@ -69,6 +69,34 @@
...
+	desc = "A simple leather covering. It's better than wearing nothing at least."
+	icon_state = "loincloth"
+	body_parts_covered = GROIN
+	can_adjust = TRUE
+	mutant_variants = NONE
+	has_sensor = NO_SENSORS
+	fitted = FEMALE_UNIFORM_TOP
+
+/obj/item/clothing/under/costume/loincloth/sensor
+	name = "loincloth"
```
Comment (Zonespace27) [inline modular_skyrat/modules/customization/modules/clothing/under/costumes.dm:85]: no need to redefine the name/desc on children if it's the same thing

File: modular_skyrat/modules/modular_vending/code/autodrobe.dm [hunk 2]
```diff
@@ -37,6 +37,8 @@
 		/obj/item/clothing/head/cowboyhat/sheriff/alt = 5,
 		/obj/item/clothing/head/cowboyhat/deputy = 5,
 		/obj/item/clothing/shoes/jackboots/timbs = 1,
+		/obj/item/clothing/under/costume/loincloth/sensor = 5,
```
Comment (Zonespace27) [inline modular_skyrat/modules/modular_vending/code/autodrobe.dm:40]: ehhhh on these being in the autodrobe but that may be me

File: code/game/objects/items/stacks/sheets/leather.dm [hunk 3]
```diff
@@ -190,6 +190,7 @@ GLOBAL_LIST_INIT(leather_recipes, list ( \
 	new/datum/stack_recipe("saddle", /obj/item/saddle, 5), \
 	//SKYRAT EDIT ADDITION BEGIN
 	new/datum/stack_recipe("gear harness", /obj/item/clothing/under/misc/gear_harness, 6), \
+	new/datum/stack_recipe("loincloth", /obj/item/clothing/under/costume/loincloth, 2), \
```
Comment (Iamgoofball) [inline code/game/objects/items/stacks/sheets/leather.dm:193]: these ones are fine but could use more shading and detailing on the sprites
chrome_pm7T7rB3Jn

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 4]
```diff
@@ -384,6 +384,7 @@ GLOBAL_LIST_INIT(cloth_recipes, list ( \
 	new/datum/stack_recipe("white jumpsuit", /obj/item/clothing/under/color/white, 3), \
 	new/datum/stack_recipe("white shoes", /obj/item/clothing/shoes/sneakers/white, 2), \
 	new/datum/stack_recipe("white scarf", /obj/item/clothing/neck/scarf, 1), \
+	new/datum/stack_recipe("loincloth", /obj/item/clothing/under/costume/loincloth/cloth, 2), /*SKYRAT EDIT ADDITION*/ \
```
Comment (Iamgoofball) [inline code/game/objects/items/stacks/sheets/sheet_types.dm:387]: but these ones still look like diapers from the front view
chrome_pm7T7rB3Jn

Comment (Iamgoofball) [reply code/game/objects/items/stacks/sheets/sheet_types.dm:387]: is there any way we can try to break up the sprite a bit to make it look less like that?

**Comment by SpaceVampire** [comment]
Please put images of the sprites and how they look on ashwalkers inside of the PR body.
Having to download the files, or look at Icon Diffs is something of a pain.

File: modular_skyrat/modules/modular_vending/code/autodrobe.dm (hunk 2 above)
Comment (weesnaw-ops) [reply modular_skyrat/modules/modular_vending/code/autodrobe.dm:40]: it was in it beforehand, so

**Comment by weesnaw-ops** [comment]
i'm going to close this theres some weird code that prevents it from compiling and, IDK what's wrong with it so

**Pull request closed without merging**